common/app/views/fragments/atoms/chart.scala.html

```diff
@@ -0,0 +1,50 @@
+@import common.InlineJs
+@import model.content.ChartAtom
```

Comment:
This looks quite similar to the `interactiveAtom` html - do we believe that chart atoms are functionally different to interactiveAtoms in that we need to use a separate view? Or are they just `interactiveAtoms` that have been typed as `chartAtoms` (in which case we can re-use interactiveAtom code where it makes sense)

Reply:
This is an excellent question. My plan is to leave them separate and see how the next atoms play out and then make the right abstraction if I can. Doing it now would feel premature optimisation for me, but happy to be proven wrong, if that is a problem.

Reply:
I think that's fair yes, I *hope* all can be treated equally and we only end up needing the one template!

Reply:
Sure!